### PR TITLE
Improvements of the HelloWorld pack

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
**Improved the HelloWorld pack to conform to our updated XSOAR standards:**
- Improved the main function by removing repetitive calls to demisto.params(), demisto.command() and demisto.args(). 
- Added to the test_module a call to the 'fetch_incidents' command.
- Improved the Unit-test docstrings to our updated docstring (given, when, then). 
- Added a unit test to increase the coverage. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py [hunk 1]
```diff
@@ -552,6 +553,23 @@ def test_module(client: Client, first_fetch_time: int) -> str:
...
+    # Tests fetch incident:
+    if params.get('isFetch'):
+        alert_status = params.get('alert_status', None)
+        alert_type = params.get('alert_type', None)
+        min_severity = params.get('min_severity', None)
+
+        fetch_incidents(
+            client=client,
+            max_results=1,
+            last_run=demisto.getLastRun(),
```
Comment (ShahafBenYakir) [inline Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py:566]:
```suggestion
            last_run={},
```

File: Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py [hunk 2]
```diff
@@ -1364,38 +1387,41 @@ def main() -> None:
             # of incidents to create
             demisto.incidents(incidents)
 
-        elif demisto.command() == 'ip':
-            default_threshold_ip = int(demisto.params().get('threshold_ip', '65'))
-            return_results(ip_reputation_command(client, demisto.args(), default_threshold_ip, reliability))
+        elif command == 'ip':
+            default_threshold_ip = int(params.get('threshold_ip', '65'))
```
Comment (ShahafBenYakir) [inline Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py:1391]:
```suggestion
            default_threshold_ip = arg_to_number(params.get('threshold_ip', '65'))
```

Comment (ShahafBenYakir) [reply Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py:1391]: Same in domain command

**Review by ShahafBenYakir — APPROVED** [review]
Great

File: Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py (hunk 2 above)
Comment (ShacharKidor) [reply Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py]: Done. Thank you!

File: Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py [hunk 3]
```diff
@@ -552,6 +553,23 @@ def test_module(client: Client, first_fetch_time: int) -> str:
             return 'Authorization Error: make sure API Key is correctly set'
         else:
             raise e
+
+    # Tests fetch incident:
+    if params.get('isFetch'):
```
[the remark this answers is not in the document]
Comment (ShacharKidor) [reply Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py]: Fixed.

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir @JudahSchwartz Doc review completed.
@ShacharKidor  I created a pack readme that I sent you via Slack. Please add it to this PR.

**Comment by ShacharKidor** [comment]
> @ShahafBenYakir @JudahSchwartz Doc review completed. @ShacharKidor I created a pack readme that I sent you via Slack. Please add it to this PR.

@ShirleyDenkberg  - Added. Thank you!

**Pull request merged**